fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -2442,11 +2432,13 @@ public void updateFragmentExecStatus(TReportExecStatusParams params) {
         if (params.isSetErrorTabletInfos()) {
             updateErrorTabletInfos(params.getErrorTabletInfos());
         }
-        if (params.isSetHivePartitionUpdates() && hivePartitionUpdateFunc != null) {
-            hivePartitionUpdateFunc.accept(params.getHivePartitionUpdates());
+        if (params.isSetHivePartitionUpdates()) {
+            ((HMSTransaction) Env.getCurrentEnv().getGlobalExternalTransactionInfoMgr().getTxnById(txnId))
```

Comment:
I think we need to check if transaction still exists.
It may already be removed before BE reporting.